docs/docker-hub/accounts.md

```diff
@@ -79,7 +79,7 @@ repository access rights to groups:
 - `Write` access users are able to push to non-automated repositories on the Docker
   Hub.
 - `Admin` access allows the user to modify the repositories "Description", "Collaborators" rights,
-  "Mark as unlisted", "Public/Private" status and "Delete".
+  "Mark as unlisted", "Public/Private" status and "Delete", and to retrieve build logs.
```

Comment:
"and to retrieve build logs" did you mean:

"Mark as unlisted", "Public/Private" status, and "Delete" to retrieve build logs.